debian/changelog

```diff
@@ -1,3 +1,9 @@
+netatalk (3.1.8-1~logrotate) experimental; urgency=low
```

Comment:
Could you please combine your changelog entry with my 3.1.8-1 entry? Especially keep the UNRELEASED, it's a common indicator that the packaging isn't official/finalised.

If you want, name it 3.1.8-2, but that's optional. You can use gbp dch (or git-dch) to easy changelog handling.

Otherwise good and ready to merge.